fjord/search/apps.py

```diff
@@ -0,0 +1,19 @@
+from django.apps import AppConfig
+from django.conf import settings
+
+from elasticsearch_dsl.connections import connections
+
+
+DEFAULT_URLS = ['localhost']
+DEFAULT_TIMEOUT = 5
+
+
+class SearchConfig(AppConfig):
+    name = 'fjord.search'
+
+    def ready(self):
+        # Set up Elasticsearch connections based on settings
+        urls = getattr(settings, 'ES_URLS', DEFAULT_URLS)
+        timeout = getattr(settings, 'ES_TIMEOUT', DEFAULT_TIMEOUT)
+
+        connections.configure(default={'hosts': urls, 'timeout': timeout})
```

Comment:
elasticsearch-dsl maintains connections. So here, we configure all the connections (in our case, it's just "default") so that we can use them elsewhere.